hikari/impl/bot.py

```diff
@@ -356,15 +358,20 @@ def _check_if_alive(self) -> None:
         if not self._is_alive:
             raise errors.ComponentStateConflictError("bot is not running so it cannot be interacted with")
 
-    async def close(self, force: bool = True) -> None:
+    async def close(self) -> None:
```

Comment:
how does this split compare with just moving most of this logic to a close method used internally and having the public close just be
```py
self._check_alive()
if self._closing_event:
    self._closing_event.set()

await self.join()
```

complexity wise?

Reply:
Yep, turns out to be much nicer, will change